v1.1-beta/matrixx-digital-commerce/matrixx-tmf-service-activation-management/cnf-testsuite.yml

```diff
@@ -0,0 +1,13 @@
+---
+helm_directory: "tmf-party-management"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values-2064780477.yaml\
```

Comment:
@deerowbear @daniel-wilmes generic please for end-users to be able to repeat testing if desired